benchmarks/filtsmooth.py

```diff
@@ -56,7 +14,8 @@ class Filtering:
     params = [[("ekf", "classic"), ("ekf", "sqrt"), ("ukf", "classic")]]
 
     def setup(self, linearization_implementation):
-        dynmod, measmod, initrv, info = load_pendulum()
+        regression_problem, statespace_components = filtsmooth_zoo.pendulum()
```

Comment:
I think it would be a good idea to set the pendulum parameters into stone here. Who knows when we will change the defaults.